src/fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -23,7 +23,9 @@
 
 # The list of file paths to the imaging (.avi) files
 ophys_file_paths = [
-    ophys_file_name for ophys_file_name in ophys_folder_path.iterdir() if ophys_file_name.suffix == ".avi"
+    ophys_file_name
+    for ophys_file_name in ophys_folder_path.iterdir()
+    if ophys_file_name.suffix == ".avi" and ophys_file_name.stem.startswith("invivo")
```

Comment:
you're right @CodyCBakerPhD, in my local setup I filter for files that start with the "invivo" prefix, they should cover all imaging data. how about this?

Reply:
Sure, that works